### PR TITLE
feat: Add audit table for GET /events

### PR DESCRIPTION
**Pull request by oskarwilliams378**

### CODE REVIEW

File: src/main/kotlin/uk/gov/gdx/datashare/repository/EventApiAudit.kt [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  val oauthClientId: String,
+  @Schema(description = "URL", required = true, example = "https://d33v84mi0vopmk.cloudfront.net/events")
+  val url: String,
+  @Schema(description = "METHOD", required = true, example = "GET")
+  val requestMethod: String,
+  @Schema(description = "Data of event(s)", required = true, example = "{\"data\":[]}")
+  val payload: Payload,
+
+  @Schema(description = "Time of call", required = true)
+  val whenCreated: LocalDateTime,
```
Comment (olevett) [inline src/main/kotlin/uk/gov/gdx/datashare/repository/EventApiAudit.kt:36]: Is this actually time of call, or is this when we save it?

Comment (oskarwilliams378) [reply src/main/kotlin/uk/gov/gdx/datashare/repository/EventApiAudit.kt:36]: Its when we save, will update the description

File: src/test/kotlin/uk/gov/gdx/datashare/controller/EventsControllerTest.kt [hunk 2]
```diff
@@ -166,36 +168,6 @@ class EventsControllerTest {
     }
   }
 
-  @Test
```
Comment (olevett) [inline src/test/kotlin/uk/gov/gdx/datashare/controller/EventsControllerTest.kt:169]: Removing this test seems a little orthogonal to the actual change, is there a reason it doesn't work?

**Review by olevett — CHANGES_REQUESTED** [review]
Couple of queries on some of the test related changes

File: src/test/kotlin/uk/gov/gdx/datashare/controller/EventsControllerTest.kt (hunk 2 above)
Comment (oskarwilliams378) [reply src/test/kotlin/uk/gov/gdx/datashare/controller/EventsControllerTest.kt:169]: Its a duplicate test, it broke when I was doing this change which is why I noticed it

**Review by olevett — APPROVED** [review]

**Pull request merged**